src/compiler/commandLineParser.ts

```diff
@@ -145,6 +145,11 @@ namespace ts {
             description: Diagnostics.Raise_error_on_expressions_and_declarations_with_an_implied_any_type,
         },
         {
+            name: "noImplicitOverride",
+            type: "boolean",
+            description: Diagnostics.Member_function_0_was_marked_override_but_no_matching_definition_was_found_in_the_superclass_chain_1,

```

Comment:
Description here is what you would see on the commandline under `-?`, i.e. this should be a string like "Raise an error when overriding class methods are not marked with 'overide'"